hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/OzoneManager.java

```diff
@@ -1673,6 +1698,18 @@ public void start() throws IOException {
 
     keyManager.start(configuration);
 
+    try {
```

Comment:
Minor nit: This can be fixed in a new PR, since `KeyManager` is dependent on `scmTopologyClient`, maybe we should start the client before the `KeyManager`

Reply:
Noted, thank you for the review @kerneltime!